resources/js/Pages/Auth/Profile.vue

```diff
@@ -0,0 +1,38 @@
+<script setup lang="ts">
+import {defineProps} from 'vue'
+import { router } from '@inertiajs/vue3'
```

Comment:
Do we really need this router? Please look here https://github.com/blumilksoftware/keating/blob/main/resources/js/Pages/Dashboard/PasswordUpdate.vue